docs/tfengine/secrets.md

```diff
@@ -0,0 +1,48 @@
+# Secrets
+
+The secrets deployment is used to define sensitive values that should not be
+set in plaintext. Secrets are deployed in the devops project.
```

Comment:
>Secrets are deployed in the devops project
Is that still true? Looks like secrets get their own "example-secrets" project now.

Reply:
Done.